docs/usextgeoroxar.rst

```diff
@@ -394,6 +394,60 @@ Get average properties per zone
     if __name__ == "__main__":
         main()
 
+Filter logs on facies/zone boundaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Petrophysical property modelling can be more precise if so-called shoulder effects
+are filtered. Here is a small example on how to do this:
+
+.. code-block:: python
+
+    import xtgeo
+
+    PRJ = project
+
+    TRAJNAME = "Drilled trajectory"
+    LRUNNAME = "log"
+    ZONELOGNAME = "Zone"
+    FACIESLOGNAME = "Facies"
+    INLOGS = [ZONELOGNAME, FACIESLOGNAME]
+    PETROLOGS = {"KLOGH": "KLOGH_orig", "PHIT": "PHIT_orig", "Sw": "Sw_orig"}
+    FILTER: {"tvd": 1.5}  # filter 1.5m below and above boundary in TVD
+
+
+    def filter_shoulder():
+        """Filter should bed data."""
+        for well in PRJ.wells:
+            wll = xtgeo.well_from_roxar(
```

Comment:
In order to improve the naming in the example, `well` -> `rms_well` and `wll` -> `xtgeo_well`?

Reply:
Good idea